widgets/hits-per-page-selector/__tests__/hits-per-page-selector-test.js

```diff
@@ -73,9 +74,11 @@ describe('hitsPerPageSelector()', () => {
       ],
       setValue: () => {}
     };
-    expect(ReactDOM.render.calledOnce).toBe(true, 'ReactDOM.render called once');
+    expect(ReactDOM.render.calledTwice).toBe(true, 'ReactDOM.render called twice');
     expect(ReactDOM.render.firstCall.args[0]).toEqualJSX(<Selector {...props} />);
     expect(ReactDOM.render.firstCall.args[1]).toEqual(container);
+    expect(ReactDOM.render.secondCall.args[0]).toEqualJSX(<Selector {...props} />);
```

Comment:
Good, in the end it was not so much boilerplate code to add to test this bug. In the future we might just have a `BaseWidget` to break a little bit this redundant logic we have (cssClasses)